xcube/core/store/descriptor.py

```diff
@@ -141,36 +169,55 @@ def __init__(self,
                          time_range=time_range,
                          time_period=time_period,
                          open_params_schema=open_params_schema)
+        self.coords = coords
         self.dims = dict(dims) if dims else None
-        self.data_vars = list(data_vars) if data_vars else None
+        if data_vars:
+            if type(data_vars) == dict:
```

Comment:
This is wrong w.r.t. type declaration. Fix:

```suggestion
            if instanceof(data_vars, Mapping):
```